pytest/lib/cluster.py

```diff
@@ -10,10 +10,15 @@
 import time
 import base58
 import base64
-import retry
-from gcloud import compute, get_zone, ssh, copy_to_instance
+from retry import retry as retry2
```

Comment:
Would be better to import as is, and change the few places where I use `retry` to `retrying`.

Reply:
Sounds good